CarService.Api/Controllers/OrdersController.cs

```diff
@@ -37,6 +37,25 @@ public OrdersController(IOrderService orderService)
             return Ok();
         }
 
+        /// <summary>Create mechanic review proposition</summary>
+        /// <returns>an IActionResult</returns>
+        /// <remarks>TODO detail description</remarks>
+        /// <param name="reviewPropositionDto">Orders' model</param> 
+        /// <response code="200">review proposition create successful</response>
+        /// <response code="401">Unauthorized user</response> 
+        /// <response code="500">Internal Server Error</response> 
+        [Authorize]
+        [HttpPost("create-review-proposition")]
+        public async Task<IActionResult> CreateReviewProposition([FromBody]ReviewPropositionCreationDto reviewPropositionDto)
+        {
+            string email = User.Identity.Name;
```

Comment:
Isn't enough the authorize attribute?

Reply:
No, it's not necessary. will be deleted in the following commit